src/librustdoc/html/render/write_shared.rs

```diff
@@ -682,7 +680,7 @@ impl TypeAliasPart {
             ));
 
             let part = OrderedJson::array_sorted(
-                impls.iter().map(OrderedJson::serialize).collect::<Result<Vec<_>, _>>().unwrap(),
+                impls.map(OrderedJson::serialize).collect::<Result<Vec<_>, _>>().unwrap(),
```

Comment:
Just FYI that this `collect` is also redundant,
this will work too:

```suggestion
                impls.map(|impl_| OrderedJson::serialize(impl_).unwrap()),
```
(though doesn't seem to affect perf on my local tests)

Reply:
Nice suggestion! I have applied it.